process/throttle/block_test.go

```diff
@@ -0,0 +1,3 @@
+package throttle_test
+
+//TODO: Unit tests should be added here
```

Comment:
Unit tests are a must before merging

Reply:
Yep, working on it. I pushed first without them for an easily code review (few numbers of lines :) )